lib/crypto.js

```diff
@@ -592,23 +592,21 @@ exports.Certificate = Certificate;
 function Certificate() {
   if (!(this instanceof Certificate))
     return new Certificate();
-
-  this._handle = new binding.Certificate();
```

Comment:
We tagged it semver-minor because it's cleanup, not a bug fix.  The only thing it does is replace the (superfluous) `_handle` private property for direct binding calls.